tests/test_expenses.py

```diff
@@ -36,3 +61,18 @@ def test_average_expenses_of_houses_by_categories(self):
         avg_expenses_by_categories = Expenses.average_expenses_of_houses_by_categories(houses=houses)
         assert avg_expenses_by_categories[0]['average'] == 200
         assert avg_expenses_by_categories[1]['average'] == 350
+
+
+@pytest.mark.django_db
+class TestExpenseForm:
+    def test_expense_form(self, generate_expense_form):
+        form = generate_expense_form
+        assert form.is_valid()
+
+    def test_bad_form_amount(self, generate_bad_amount_expense_form):
+        form = generate_bad_amount_expense_form
+        assert not form.is_valid()
+
+    def test_bad_form_description(self, generate_bad_description_expense_form):
+        form = generate_bad_description_expense_form
+        assert not form.is_valid()
```

Comment:
sorry, it is redundant, didn't notice the checks are in this file.
please remove

Reply:
fixed